src/wazuh_modules/vulnerability_scanner/tests/unit/cve5FB_test.cpp

```diff
@@ -0,0 +1,241 @@
+/*
+ * Wazuh SyscollectorFlatbuffers
```

Comment:
Shouldn't this be ... ?
```suggestion
 * Wazuh Vulnerability Scanner - Unit Tests
```